src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Color[] colors = Color.values();
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(colors.length);
+        return colors[index].name().toLowerCase();
```

Comment:
Returning the color in lowercase might not be consistent with other parts of the code that expect the enum constant format. Consider returning the enum constant name as is, without altering its case.